java-connectors/kafka-connect-azure-servicebus/src/test/java/io/lenses/streamreactor/connect/azure/servicebus/util/KcqlConfigBusMapperTest.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright 2017-2024 Lenses.io Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.lenses.streamreactor.connect.azure.servicebus.util;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.apache.kafka.common.config.ConfigException;
+import org.junit.jupiter.api.Test;
+
+class KcqlConfigBusMapperTest {
+
+  private static final String ASTERISK = "'";
+  private static final String EQUALS_SIGN = "=";
+  private static final String COMMA = ",";
+  private static final String PROPERTIES_BEGINNING = "PROPERTIES (";
+  private static final String PROPERTIES_ENDING = ")";
+  private static final String SUBSCRIPTION_NAME = "SUBSCRIPTION1";
+  private static final String TOPIC_TYPE = "TOPIC";
+
+  @Test
+  void mapInputToOutputsFromConfigForMultipleKcqlStatementsShouldReturnMapOfInputToOutput() {
+    //given
+    int numberOfMappings = 3;
+    List<String> inputs = new ArrayList<>(numberOfMappings);
+    List<String> outputs = new ArrayList<>(numberOfMappings);
+    String kcqlTemplate = "insert into %s select * from %s";
+    StringBuilder fullKcql = new StringBuilder();
+
+    for (int i = 0; i < numberOfMappings; i++) {
+      String newInput = "INPUT" + i;
+      String newOutput = "OUTPUT" + i;
+
+      inputs.add(i, newInput);
+      outputs.add(i, newOutput);
+      fullKcql.append(String.format(kcqlTemplate, newOutput, newInput));
+      fullKcql.append(" " + createNecessaryPropertiesPart() + ";");
```

Comment:
As above, let's not mix StringBuilder and string concatenation.

Reply:
yup!